src/cmd/linuxkit/pkg_build.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/linuxkit/linuxkit/src/cmd/linuxkit/pkgsrc"
```

Comment:
`pkgsrc` is confusingly overloaded term to some of us...

Reply:
I struggled with the name since `pkg` also means something in Go directory structures and `package` is a keyword. I'm open to suggestions.

Reply:
maybe `pkglib`?

Reply:
Sure.

Reply:
I think that goes best with `Pkg` rather than `PkgLib` for the type so I shall do so.